inference-engine/ie_bridges/c/src/ie_c_api.cpp

```diff
@@ -1698,3 +1712,94 @@ void ie_blob_free(ie_blob_t **blob) {
         *blob = NULL;
     }
 }
+
+IEStatusCode ie_blob_rwmap(const ie_blob_t *blob, ie_blob_map_t *blob_rwmap) {
+    if (blob == nullptr || blob_rwmap == nullptr) {
+        return IEStatusCode::GENERAL_ERROR;
+    }
+
+    try {
+        IE::MemoryBlob::Ptr memory_blob = IE::as<IE::MemoryBlob>(blob->object);
+        if (memory_blob) {
+            std::unique_ptr<ie_locked_memory_t> rw_locked_memory (new ie_locked_memory_t);
+            rw_locked_memory->rw_object = new IE::LockedMemory<void> (memory_blob->rwmap());
```

Comment:
why do we need to copy `LockedMemory`? I suppose that in dtor of `memory_blob->rwmap()` the memory will be unmapped. I think we need to store original `LockedMemory` returned by `memory_blob->rwmap()`

Reply:
I guess we need to modify `LockedMemory` class as well to support "move" semantic and use it here.